docs/develop/overview-authn-authz.md

```diff
@@ -32,7 +32,7 @@ You can get authorization to Microsoft Graph data for your add-in by obtaining a
 
 ## User authentication with SSO
 
-To use SSO to authenticate the user, your code in a task pane or function file calls the [getAccessToken](/javascript/api/office/officeruntime.auth#getAccessToken-options--callback-) method. If the user is not signed into Office, Office will open a dialog and navigate it to the Azure Active Directory login page. After the user is signed in, or if the user is already signed in, the method returns an access token. The token is a bootstrap token in the **On Behalf Of** flow. (See [Access to Microsoft Graph with SSO](#access-to-microsoft-graph-with-sso).) However, it can be used as an ID token as well, because it contains several claims that are unique to the current user, including `preferred_username`, `name`, `sub`, and `oid`. For guidance on which property to use as the ultimate user ID, see [Microsoft identity platform access tokens](https://docs.microsoft.com/azure/active-directory/develop/access-tokens#payload-claims). For an example of a one of these tokens, see the [Example access token](sso-in-office-add-ins.md#example-access-token).
+To use SSO to authenticate the user, your code in a task pane or function file calls the [getAccessToken](/javascript/api/office-runtime/officeruntime.auth#getAccessToken-options-) method. If the user is not signed into Office, Office will open a dialog and navigate it to the Azure Active Directory login page. After the user is signed in, or if the user is already signed in, the method returns an access token. The token is a bootstrap token in the **On Behalf Of** flow. (See [Access to Microsoft Graph with SSO](#access-to-microsoft-graph-with-sso).) However, it can be used as an ID token as well, because it contains several claims that are unique to the current user, including `preferred_username`, `name`, `sub`, and `oid`. For guidance on which property to use as the ultimate user ID, see [Microsoft identity platform access tokens](https://docs.microsoft.com/azure/active-directory/develop/access-tokens#payload-claims). For an example of a one of these tokens, see the [Example access token](sso-in-office-add-ins.md#example-access-token).
```

Comment:
The bookmark portion of the link target `/javascript/api/office-runtime/officeruntime.auth#getAccessToken-options-)` needs to be all lowercase in order for the bookmark to work properly.

Reply:
fixed